src/rules.js

```diff
@@ -1841,6 +1841,39 @@ module.exports = [
     },
     /**
      ==================================================================
+     TEXT-INDENT
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        name: 'Text indent',
+        prefix: 'Ti-',
+        properties: ['text-indent'],
+        allowCustom: true,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'inh', values: ['inherit']}
+        ]
+    },
+    /**
+     ==================================================================
+     TEXT-OVERFLOW
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        name: 'Text overflow',
+        prefix: 'Tov-',
+        properties: ['text-overflow'],
+        allowCustom: true,
+        allowSuffixToValue: false,
+        rules: [
+            {suffix: 'c', values: ['clip']},
```

Comment:
People should be able to do that via a custom value for that property. If I hadn't disable `allowSuffixToValue` they could even pass it as the value identifier, for example: `Tov-c_e`, but I think it's too dangerous.